engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/job/preparation/JobPreparationActor.scala

```diff
@@ -149,9 +149,9 @@ class JobPreparationActor(workflowDescriptor: EngineWorkflowDescriptor,
       case Success(_: DockerImageIdentifierWithoutHash) if !hasDockerDefinition => 
         lookupKvsOrBuildDescriptorAndStop(inputs, attributes, NoDocker, None)
 
-      // Even if the docker image has a hash, we need to (try to) find out the size, so send a request
-      case Success(dockerImageId: DockerImageIdentifierWithHash) =>
-        sendDockerRequest(dockerImageId)
+      // If there's a digest, we'll skip looking up the container because it just looks up the compressed size
```

Comment:
Even if a docker image hash is provided, Cromwell issues the Docker request to get the docker image size for [adjusting boot disk size](https://github.com/broadinstitute/cromwell/blob/develop/supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala#L185) on the Google backend. So this behavior could perhaps be a configuration opt-in but I don't think it should be the default.